pass.go

```diff
@@ -127,27 +127,44 @@ func (k *passKeyring) itemExists(key string) bool {
 }
 
 func (k *passKeyring) Keys() ([]string, error) {
-	var keys = []string{}
-	var path = filepath.Join(k.dir, k.prefix)
+	var (
+		keys  = []string{}
+		path  = filepath.Join(k.dir, k.prefix)
+		rpath string
+	)
 
-	info, err := os.Stat(path)
+	info, err := os.Lstat(path)
```

Comment:
@stefancocora Can't we use the `pass ls` command for listing keys instead of iterating directories?